src/golang/lib/job/spec.go

```diff
@@ -144,6 +145,14 @@ type ExtractSpec struct {
 	OutputMetadataPath string   `json:"output_metadata_path"  yaml:"output_metadata_path"`
 }
 
+type DeleteSavedObjectsSpec struct {
+	BasePythonSpec
+	ConnectorName     map[string]integration.Service `json:"connector_name"  yaml:"connector_name"`
+	ConnectorConfig   map[string]auth.Config         `json:"connector_config"  yaml:"connector_config"`
+	Parameters        map[string][]string            `json:"parameters"  yaml:"parameters"`
```

Comment:
Unlike workflow spec where `Parameters` actually stand for workflow parameters, using `Parameters` here might be a bit vague. We could be more specific here by using something like `IntegrationToObjectMap`.

Reply:
Updated `Parameters`/`parameters` to `IntegrationToObject`/`integration_to_object`!